ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/Common/SPNativeScreenViewController.swift

```diff
@@ -39,14 +39,22 @@ enum SPUIRectEdge {
 }
 
 extension UIFont {
-    convenience init?(from spFont: SPNativeFont?) {
+    convenience init?(from spFont: SPNativeFont?, adjust: UITextView? = nil) {
         let magicScalingFactor = CGFloat(1.8)
-        let fontSize = spFont?.fontSize != nil ? spFont!.fontSize * magicScalingFactor : UIFont.preferredFont(forTextStyle: .body).pointSize
+        var fontSize = spFont?.fontSize != nil ? spFont!.fontSize * magicScalingFactor : UIFont.preferredFont(forTextStyle: .body).pointSize
         let family = spFont?.fontFamily
             .split(separator: ",")
             .map { $0.trimmingCharacters(in: .whitespacesAndNewlines).lowercased() }
             .first { UIFont.familyNames.map { $0.lowercased() }.contains($0) }
             ?? UIFont.systemFont(ofSize: fontSize).familyName
+        if let adjust = adjust {
+            adjust.font = UIFont(name: family, size: fontSize)
+            while (adjust.contentSize.height > adjust.frame.size.height) {
+                print(adjust.contentSize.height.description + ">" + adjust.frame.size.height.description)
```

Comment:
don't forget to remove `print` statements

Reply:
Oops. Fixed